### PR TITLE
Css refactor

### PR DESCRIPTION
**Pull request by ZadenRB**

### CODE REVIEW

File: src/components/NavMenuItems.tsx [hunk 1]
```diff
@@ -1,59 +1,127 @@
...
-          <strong>Donate</strong>
-        </Button>
-      </Menu.Item>
-    </>
-  );
+import { NavHashLink } from 'react-router-hash-link';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBars } from '@fortawesome/free-solid-svg-icons';
+
+export class NavMenuItems extends React.Component {
```
Comment (dviramontes) [inline src/components/NavMenuItems.tsx]: We shouldn't need to turn this into a class component to add life cycle events. 
React hooks is a better API for this sort of thing. [Checkout this guide.](https://reactjs.org/docs/hooks-intro.html) The video by abramov is how i learned to use them. Highly recommend it.

Comment (ZadenRB) [reply src/components/NavMenuItems.tsx]: Thanks for the suggestion, implemented 👍

File: src/pages/Volunteer.tsx [hunk 2]
```diff
@@ -1,189 +1,151 @@
...
-                  />
-                </div>
+            </span>
+          </div>
+          <div>
+            <div className="iframe-parent">
+              <div className="iframe-container">
+                <iframe
+                  title="Code for Denver meeting location"
+                  src={`https://www.google.com/maps/embed/v1/place?key=${process.env.REACT_APP_GOOGLE_MAPS_API_KEY}&q=Galvanize+-+Denver+(Platte)`}
```
Comment (dviramontes) [inline src/pages/Volunteer.tsx]: can we update this location to workability ?

Comment (ZadenRB) [reply src/pages/Volunteer.tsx]: Updated!

**Pull request merged**